hue/hue/base/hive-site-xml-secret.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hue-hive-site-xml
+type: Opaque
+stringData:
+  hive-site.xml: |
+    <?xml version="1.0"?>
+    <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+    <configuration>
+      <property>
+        <name>hive.server2.transport.mode</name>
+        <value>binary</value>
+        <description>Server transport mode. binary or http.</description>
+      </property>
+
+      <property>
+        <name>hive.server2.thrift.http.port</name>
+        <value>10000</value>
+        <description>Port number when in HTTP mode.</description>
+      </property>
+
+      <property>
+        <name>fs.s3a.awsAccessKeyId</name>
+        <value></value>
```

Comment:
Shouldn't this line use a variable to populate the S3 Access Key

Reply:
@rimolive all requested changes should be fixed now, please check again :+1: